src/appMain/CMakeLists.txt

```diff
@@ -143,6 +143,12 @@ endif()
 add_executable(${PROJECT} ${SOURCES})
 target_link_libraries(${PROJECT} ${LIBRARIES})
 
+add_dependencies(${PROJECT} Policy)
+
+if(ENABLE_LOG)
+  add_dependencies(${PROJECT} install-3rd_party_logger)
```

Comment:
@istoilovgithub appMain depends from Utils and must wait building of Utils to finish(and all its dependencies). Utils must take care of logger - build/install. Is it needed to add here explicit dependency of logger install?